config/internal/common/default/mlmd-envoy-dashboard-access-policy.yaml.tmpl

```diff
@@ -11,7 +11,7 @@ spec:
   ingress:
     - ports:
         - protocol: TCP
-          port: 9090
+          port: 8443
```

Comment:
so currently this NP is saying "for all traffic coming for 8443, only allow traffic from pods that have label "app: odh-dashboard" in ANY namespace, or pods that have the label "component: data-science-pipelines" in the NP's own namespace

we don't want this behavior, what we want is: 

* block any traffic to port 9090 
* allow traffic from ANYWHERE to port 8443 (oauth proxy) 

something like this: 

```yaml
spec:
  podSelector:
    matchLabels:
      app: ds-pipeline-metadata-envoy-sample
      component: data-science-pipelines
  ingress:
    - ports:
        - protocol: TCP
          port: 8443
    - ports:
        - protocol: TCP
          port: 9090
      from:
        - podSelector:
            matchLabels:
              component: data-science-pipelines # it's fine if the dspa pods in the same ns can access it, though they don't need to, I think we need to have at least one rule 
  policyTypes:
    - Ingress
```